shotover-proxy/src/transforms/cassandra/connection.rs

```diff
@@ -89,7 +90,7 @@ async fn tx_process<C: CodecWriteHalf, T: AsyncWrite>(
     codec: C,
 ) -> Result<()> {
     let in_w = FramedWrite::new(write, codec);
-    let rx_stream = UnboundedReceiverStream::new(out_rx).map(|x| {
+    let rx_stream = TokioStreamExt::map(UnboundedReceiverStream::new(out_rx), |x| {
```

Comment:
Are these `TokioStreamExt::map` changes still needed?
From memory these were due to issues with crate version mismatches?

Reply:
I get a "multiple map found error"

Reply:
I reverted the entire file to origin/main and then reintroduced `use crate::concurrency::FuturesOrdered;` -> `use futures::stream::FuturesOrdered;` and it compiles fine.

Same with server.rs

Reply:
Cool, I'll fix that up. Thanks